config/deploy/staging.rb

```diff
@@ -0,0 +1,23 @@
+############################################
+# Staging Server
+############################################
+
+set :stage, :staging
+set :stage_domain, "chappress-staging.chapman.edu"
+server "cprs-pre-wb01.chapman.edu", user: "wimops", roles: %w(web app db)
+set :deploy_to, "/usr/share/nginx/html"
```

Comment:
@tatwell [Update]: Switched root to `/var/www/html`. https site should be working. Added commit.